spring-boot/admin/src/main/resources/application.yml

```diff
@@ -24,4 +24,11 @@ ygg.proxy.executable-jar.filename-original-prefix: prefix
 ygg.proxy.executable-jar.filename-custom.prefix: prefix
 ygg.proxy.executable-jar.filename-custom.suffix: suffix
 ygg.proxy.generate-bootstrap-properties.spring.application.name.prefix: prefix
-ygg.proxy.generate-bootstrap-properties.spring.cloud.config.uri: http://localhost
\ No newline at end of file
+ygg.proxy.generate-bootstrap-properties.spring.cloud.config.uri: http://localhost
+
+# The Gateway Health Monitor updates the status (Ok, Warning and Critical) of all gateways based on a threshold value in minutes.
+# This threshold values allows the gateway health monitor to put the correct status based on how much time ago the last successful
+# call was made from the gateway to the admin.
+gateway_health_monitor.warning.threshold.minutes: 5
```

Comment:
please stick to the convention to use `ygg` as prefix for attributes, and use dashes instead of underscores.

Reply:
Done